tests/core/component.test.js

```diff
@@ -585,6 +585,19 @@ suite('Component', function () {
       component.updateProperties({list: ['b']});
       sinon.assert.calledOnce(updateStub);
     });
+
+    test('emit componentchanged when update calls setAttribute', function (done) {
+      var TestComponent = registerComponent('dummy', {
```

Comment:
Can name this component `test` so it gets automatically cleaned up by the test suite.